views/add_task.handlebars

```diff
@@ -1,12 +1,25 @@
-<main>
+<head>
+    <style>
+        label {
+        padding: 12px 12px 12px 0;
+        display: inline-block;
+    }
+
+    </style>
+</head>
+
+<body>
+    <main>
     <div>
         <h1>Add Task to {{project.name}}</h1>
     </div>
-    <form action='/project_board/{{project.id}}/add_task' method="POST">
+    <form class="project_boardgit"

```

Comment:
should there be a git at the end of "project_boardgit"? I can't see a class called that

Reply:
oh I don't know. I didn't do that.